manager/api/tests/test_jira.py

```diff
@@ -319,6 +345,10 @@ def test_add_comment(self):
         response.status_code = 201
         mock_jira_client.return_value = response
 
+        mock_timeloss_patcher = patch("manager.utils.update_time_loss")
+        mock_timeloss_client = mock_timeloss_patcher.start()
+        mock_timeloss_client.return_value = response
```

Comment:
I left a comment before in regards of returning a 400 response in the case the `update_time_loss` fails (on the `jira_comment` function). If you are willing to add that suggestion, that should be tested here (ensuring we get the bad response arranged by the `update_time_loss` function, in this case mocked up).